src/blocks/Hero/Hero.js

```diff
@@ -48,9 +48,12 @@ const HeroHeading = styled('h1', {
   name: 'Hero',
   slot: 'Heading',
 })(({ theme }) => ({
-  ...theme.typography.h3,
+  ...theme.typography.h2,
```

Comment:
could be interesting to look up how to overwrite  `theme.typography.h2` in `src/components/styles/createTheme.js` https://mui.com/material-ui/customization/theming/ if we want to have specific styling that applies to all h2. Everything added there will be spread when using the spread operator `...theme.typography.h2`

mui docs, there are alot of other good examples too: https://mui.com/material-ui/customization/theming/